tests/suites/caasadmission/admission.sh

```diff
@@ -0,0 +1,76 @@
+
+run_deploy_microk8s() {
+  echo
+
+  name=$1
+  export BOOTSTRAP_PROVIDER=microk8s
+  bootstrap microk8s ${name}
+
+  microk8s.config > ${TEST_DIR}/kube.conf
+}
+
+test_deploy_admission() {
+  name=test-$(petname)
+
+  run_deploy_microk8s $name
```

Comment:
We need to quote everything...

```suggestion
  run_deploy_microk8s "${name}"
```